### PR TITLE
remove noop

### PR DESCRIPTION
**Pull request by snreynolds**
## Related Issue
Which issue does this pull request resolve?

## Description of changes

### CODE REVIEW

File: test/Hooks.t.sol [hunk 1]
```diff
@@ -25,11 +25,12 @@ contract HooksTest is Test, Deployers, GasSnapshot {
     using PoolIdLibrary for PoolKey;
     using Hooks for IHooks;
 
-    address payable ALL_HOOKS_ADDRESS = payable(0xfFf0000000000000000000000000000000000000);
+    /// 1111 1111 1100
+    address payable ALL_HOOKS_ADDRESS = payable(0xFfC0000000000000000000000000000000000000);
     MockHooks mockHooks;
 
     // Update this value when you add a new hook flag. And then update all appropriate asserts.
-    uint256 hookPermissionCount = 12;
```
Comment (hensha256) [inline test/Hooks.t.sol:32]: Why was it 12? 🤔 was it just out of date before?

File: test/Hooks.t.sol [hunk 2]
```diff
@@ -91,10 +92,10 @@ contract HooksTest is Test, Deployers, GasSnapshot {
         assertEq(mockHooks.afterAddLiquidityData(), new bytes(111));
 
         modifyLiquidityRouter.modifyLiquidity(key, IPoolManager.ModifyLiquidityParams(0, 60, 0), new bytes(222));
-        assertEq(mockHooks.beforeAddLiquidityData(), new bytes(111));
-        assertEq(mockHooks.afterAddLiquidityData(), new bytes(111));
-        assertEq(mockHooks.beforeRemoveLiquidityData(), new bytes(222));
-        assertEq(mockHooks.afterRemoveLiquidityData(), new bytes(222));
+        // assertEq(mockHooks.beforeAddLiquidityData(), new bytes(111));
```
Comment (hensha256) [inline test/Hooks.t.sol:95]: Why are these commented out?

File: .forge-snapshots/poolManager bytecode size.snap [hunk 3]
```diff
@@ -1 +1 @@
-23847
\ No newline at end of file
+23425
```
Comment (hensha256) [inline .forge-snapshots/poolManager bytecode size.snap:1]: Wow I thought it would go down by more than 400 tbh

**Review by hensha256 — COMMENTED** [review]
Nice PR! Just like 1 small thing then I can confirm!

File: test/Hooks.t.sol (hunk 1 above)
Comment (snreynolds) [reply test/Hooks.t.sol:32]: I think it was out of date. Im guessing when we merged access lock this didn't get updated

File: test/Hooks.t.sol (hunk 2 above)
Comment (snreynolds) [reply test/Hooks.t.sol:95]: accident - good catch

File: .forge-snapshots/poolManager bytecode size.snap (hunk 3 above)
Comment (snreynolds) [reply .forge-snapshots/poolManager bytecode size.snap:1]: 🤷‍♀️

**Review by hensha256 — APPROVED** [review]
lgtm

**Pull request merged**